### ISSUE TITLE
Error when receive email 

### ISSUE THREAD

**Issue by vlacour97** [postalserver/postal#2412]
## Describe the bug

I have an error when i send email to postal from outlook. The mail is received but not traited in postal

## To Reproduce

1. Create namespace and mail server
2. Config email with good DNS
3. Test send email => OK
4. Add Route for receiving email
5. Send email to the routed email from outlook

## Expected behaviour

The stmp throw an error like this
```
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.612] DEBUG -- : [EKAAYD] Not logging further message data.
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.652] ERROR -- : [EKAAYD] An error occurred while processing data from a client.
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.652] ERROR -- : [EKAAYD] NameError: uninitialized constant Mail::Encodings::RubyVer
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.653] ERROR -- : [EKAAYD] /opt/postal/app/config/initializers/mail_extensions.rb:9:in `q_value_decode'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.653] ERROR -- : [EKAAYD] /usr/local/bundle/gems/mail-2.8.1/lib/mail/encodings.rb:133:in `block (2 levels) in value_decode'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.653] ERROR -- : [EKAAYD] /usr/local/bundle/gems/mail-2.8.1/lib/mail/encodings.rb:130:in `gsub!'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.653] ERROR -- : [EKAAYD] /usr/local/bundle/gems/mail-2.8.1/lib/mail/encodings.rb:130:in `block in value_decode'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.653] ERROR -- : [EKAAYD] /usr/local/bundle/gems/mail-2.8.1/lib/mail/encodings.rb:129:in `each'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.653] ERROR -- : [EKAAYD] /usr/local/bundle/gems/mail-2.8.1/lib/mail/encodings.rb:129:in `value_decode'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.653] ERROR -- : [EKAAYD] /usr/local/bundle/gems/mail-2.8.1/lib/mail/encodings.rb:108:in `decode_encode'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.653] ERROR -- : [EKAAYD] /usr/local/bundle/gems/mail-2.8.1/lib/mail/fields/unstructured_field.rb:55:in `do_decode'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.653] ERROR -- : [EKAAYD] /usr/local/bundle/gems/mail-2.8.1/lib/mail/fields/common_field.rb:55:in `decoded'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.653] ERROR -- : [EKAAYD] /usr/local/bundle/gems/mail-2.8.1/lib/mail/field.rb:233:in `method_missing'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.654] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/message_db/message.rb:324:in `block in headers'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.654] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/message_db/message.rb:321:in `each'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.654] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/message_db/message.rb:321:in `each_with_object'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.654] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/message_db/message.rb:321:in `headers'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.654] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/message_db/message.rb:85:in `copy_attributes_from_raw_message'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.654] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/message_db/message.rb:276:in `save_raw_message'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.654] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/message_db/message.rb:197:in `save'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.654] ERROR -- : [EKAAYD] /opt/postal/app/app/models/route.rb:143:in `create_messages'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.654] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/smtp_server/client.rb:491:in `block in finished'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.654] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/smtp_server/client.rb:453:in `each'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.654] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/smtp_server/client.rb:453:in `finished'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.655] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/smtp_server/client.rb:390:in `block in data'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.655] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/smtp_server/client.rb:49:in `handle'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.655] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/smtp_server/server.rb:184:in `block (2 levels) in run_event_loop'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.656] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/smtp_server/server.rb:86:in `select'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.656] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/smtp_server/server.rb:86:in `block in run_event_loop'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.656] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/smtp_server/server.rb:84:in `loop'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.656] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/smtp_server/server.rb:84:in `run_event_loop'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.657] ERROR -- : [EKAAYD] /opt/postal/app/lib/postal/smtp_server/server.rb:287:in `run'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.657] ERROR -- : [EKAAYD] /opt/postal/app/lib/tasks/postal.rake:15:in `block (2 levels) in <top (required)>'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.657] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/task.rb:281:in `block in execute'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.657] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/task.rb:281:in `each'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.657] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/task.rb:281:in `execute'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.657] ERROR -- : [EKAAYD] /usr/local/bundle/gems/sentry-ruby-5.8.0/lib/sentry/rake.rb:24:in `execute'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.657] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/task.rb:219:in `block in invoke_with_call_chain'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.657] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/task.rb:199:in `synchronize'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.657] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/task.rb:199:in `invoke_with_call_chain'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.657] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/task.rb:188:in `invoke'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.657] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/application.rb:160:in `invoke_task'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.657] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/application.rb:116:in `block (2 levels) in top_level'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.658] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/application.rb:116:in `each'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.658] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/application.rb:116:in `block in top_level'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.658] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/application.rb:125:in `run_with_threads'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.658] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/application.rb:110:in `top_level'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.658] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/application.rb:83:in `block in run'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.658] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/application.rb:186:in `standard_exception_handling'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.658] ERROR -- : [EKAAYD] /usr/local/lib/ruby/gems/3.2.0/gems/rake-13.0.6/lib/rake/application.rb:80:in `run'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.658] ERROR -- : [EKAAYD] /opt/postal/app/bin/rake:4:in `<top (required)>'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.658] ERROR -- : [EKAAYD] /usr/local/bundle/gems/bundler-2.4.9/lib/bundler/cli/exec.rb:58:in `load'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.658] ERROR -- : [EKAAYD] /usr/local/bundle/gems/bundler-2.4.9/lib/bundler/cli/exec.rb:58:in `kernel_load'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.658] ERROR -- : [EKAAYD] /usr/local/bundle/gems/bundler-2.4.9/lib/bundler/cli/exec.rb:23:in `run'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.658] ERROR -- : [EKAAYD] /usr/local/bundle/gems/bundler-2.4.9/lib/bundler/cli.rb:492:in `exec'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.659] ERROR -- : [EKAAYD] /usr/local/bundle/gems/bundler-2.4.9/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.659] ERROR -- : [EKAAYD] /usr/local/bundle/gems/bundler-2.4.9/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.659] ERROR -- : [EKAAYD] /usr/local/bundle/gems/bundler-2.4.9/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.659] ERROR -- : [EKAAYD] /usr/local/bundle/gems/bundler-2.4.9/lib/bundler/cli.rb:34:in `dispatch'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.659] ERROR -- : [EKAAYD] /usr/local/bundle/gems/bundler-2.4.9/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.659] ERROR -- : [EKAAYD] /usr/local/bundle/gems/bundler-2.4.9/lib/bundler/cli.rb:28:in `start'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.659] ERROR -- : [EKAAYD] /usr/local/bundle/gems/bundler-2.4.9/exe/bundle:45:in `block in <top (required)>'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.659] ERROR -- : [EKAAYD] /usr/local/bundle/gems/bundler-2.4.9/lib/bundler/friendly_errors.rb:117:in `with_friendly_errors'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.659] ERROR -- : [EKAAYD] /usr/local/bundle/gems/bundler-2.4.9/exe/bundle:33:in `<top (required)>'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.659] ERROR -- : [EKAAYD] /opt/postal/app/bin/bundle:3:in `load'
17/04/2023 10:48:10[36] [2023-04-17T08:48:10.660] ERROR -- : [EKAAYD] /opt/postal/app/bin/bundle:3:in `<main>'
```

## Environment details 

docker image 2.1.2

**Comment by vlacour97** [comment]
Linked to the commit: https://github.com/mikel/mail/pull/1478/commits/a58bf11afe170524817889b92a55531901c9bc9e

**Comment by catphish** [comment]
Thank you for reporting this. I have fixed this in 5998bf376a274df19f29877e7f68ea75f298c9f9
There will likely be a release of Postal today containing this fix.

**Comment by catphish** [comment]
Postal 2.1.4 has now been released with this fix.
